CMakeLists.txt

```diff
@@ -20,8 +20,10 @@ add_library(${PROJECT_NAME} ${SRC})
 target_include_directories(${PROJECT_NAME} INTERFACE src)
 
 # test
-enable_testing()
-add_subdirectory(test)
-find_package(Sanitizers)
-add_sanitizers(${PROJECT_NAME} test.exe)
-add_sanitizers(${PROJECT_NAME} test_miniz.exe)
+if (NOT KUBA_ZIP_DISABLE_TESTS)
```

Comment:
Maybe `CMAKE_DISABLE_TESTING`

Reply:
But that may clash with names of other projects...

Reply:
But if someone set `CMAKE_DISABLE_TESTING`, I suppose the intention was to disable all ctests, so also this one.

Reply:
Okay, I changed it. You can squash merge now if you want to.